pages/index.tsx

```
@@ -1,40 +1,19 @@
-// This is a skeleton starter React page generated by Plasmic.
-// This file is owned by you, feel free to edit as you see fit.
 import * as React from "react";
-import { PageParamsProvider as PageParamsProvider__ } from "@plasmicapp/react-web/lib/host";
+import * as ph from "@plasmicapp/react-web/lib/host";
 
-import { ThemeContext } from "../components/plasmic/theme_switcher/PlasmicGlobalVariant__Theme";
+import ScreenVariantProvider from "../components/plasmic/theme_switcher/PlasmicGlobalVariant__Screen";
```

ESLint

[error] 'ScreenVariantProvider' is defined but never used.

(@typescript-eslint/no-unused-vars)